src/cnn_feature_extractor/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+project(smqtk_cnn_feature_extractor)
+cmake_minimum_required(VERSION 2.8)
+
+find_package(Caffe REQUIRED)
+include_directories(SYSTEM ${Caffe_INCLUDE_DIRS})
```

Comment:
I think we're also missing a call to ``add_definitions(${Caffe_DEFINITIONS})`` here. I'm getting the issue where the ``-DCPU_ONLY`` flag is not being propagated from Caffe. Adding this line fixes the issue.